example.yml

```diff
@@ -1,4 +1,4 @@
-# yaml-language-server: $schema=https://raw.githubusercontent.com/jskherman/cv.typ/main/cv.typ.schema.json
+# yaml-language-server: $schema=./cv.typ.schema.json
```

Comment:
Could we also return the original URL here please? I don't think this will work with submodules.

Reply:
Sorry I forgot to revert it when I debugging the schema.